setup.py

```diff
@@ -432,6 +461,30 @@ def check_file(f):
         libs += ['caffe2']
         build_libs(libs)
 
+        shutil.copyfile('aten/src/THNN/generic/THNN.h', 'torch/lib/THNN.h')
```

Comment:
What's the reasoning for for still having build_pytorch_libs.sh and setup.py? How are we deciding which file to put logic in?

Reply:
I'm working on completely absorbing build_pytorch_libs.sh into setup.py. This PR is kind of a miscellaneous subset of that effort

Reply:
Great idea. In some ways I'd like to have build_pytorch_libs.sh absorbed into CMakeLists.txt, since the entry point for libtorch builds is different from setup.py builds. Could we go that direction instead?

Reply:
good point, I'll move these couple lines into cmake somewhere. Do you think it's worth figuring out how to move the caffe2/cpp_test bit (just below) as well? As far as I know they're only used in the context of setup.py builds

Reply:
You're right that they are only used in setup.py builds (for CI testing) but that might change in the future. Ideally, if it's quick and easy, it'd likely be best to bring them into cmake as well. Only if it is easy, though. Otherwise, feel free to leave those here. Thank you for all of this - great work!